community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java

```diff
@@ -177,22 +180,22 @@ TransactionType upgradeToSchemaTransaction() throws InvalidTransactionTypeKernel
     private CloseListener closeListener;
 
     public KernelTransactionImplementation( StatementOperationParts operations,
-            SchemaWriteGuard schemaWriteGuard, LabelScanStore labelScanStore,
-            IndexingService indexService,
-            UpdateableSchemaState schemaState,
-            TransactionRecordState recordState,
-            SchemaIndexProviderMap providerMap, NeoStore neoStore,
-            Locks.Client locks, TransactionHooks hooks,
-            ConstraintIndexCreator constraintIndexCreator,
-            TransactionHeaderInformationFactory headerInformationFactory,
-            TransactionCommitProcess commitProcess,
-            TransactionMonitor transactionMonitor,
-            StoreReadLayer storeLayer,
-            LegacyIndexTransactionState legacyIndexTransactionState,
-            Pool<KernelTransactionImplementation> pool,
-            ConstraintSemantics constraintSemantics,
-            Clock clock,
-            TransactionTracer tracer )
+                                            SchemaWriteGuard schemaWriteGuard, LabelScanStore labelScanStore,
```

Comment:
Cool - we'll send off another PR (I hope) later today that cleans this up then, if that's cool? That way we can merge this and start on the next piece without waiting for the build :)